pkg/flags/strings.go

```diff
@@ -22,6 +22,20 @@ func NewStringsFlag(valids ...string) *StringsFlag {
 	return &StringsFlag{Values: valids}
```

Comment:
I think it's OK to panic in that case; it shouldn't be called without any valid strings.